src/Sulu/Bundle/AdminBundle/Tests/Unit/FormMetadata/FormXmlLoaderTest.php

```diff
@@ -26,57 +28,61 @@ class FormXmlLoaderTest extends TestCase
      */
     private $loader;
 
-    /**
-     * @var ExpressionLanguage
-     */
-    private $expressionLanguage;
-
     /**
      * @var TranslatorInterface
      */
     private $translator;
 
     public function setUp()
     {
-        $this->expressionLanguage = $this->prophesize(ExpressionLanguage::class);
         $this->translator = $this->prophesize(TranslatorInterface::class);
         $propertiesXmlParser = new PropertiesXmlParser(
             $this->translator->reveal(),
             ['en' => 'en', 'de' => 'de', 'fr' => 'fr', 'nl' => 'nl']
         );
         $schemaXmlParser = new SchemaXmlParser();
-        $this->loader = new FormXmlLoader($propertiesXmlParser, $schemaXmlParser);
+        $locales = ['de', 'en'];
+        $formMetadataMapper = new FormMetadataMapper('../../Application/var/cache', true);
+        $this->loader = new FormXmlLoader($propertiesXmlParser, $schemaXmlParser, $locales, $formMetadataMapper);
     }
 
     public function testLoadForm()
     {
-        /** @var FormMetadata $formMetadata */
-        $formMetadata = $this->loader->load($this->getFormDirectory() . 'form.xml');
+        /** @var LocalizedFormMetadataCollection $formMetadataCollection */
```

Comment:
Shouldn't most of the comments like this in this file be superfluous? Can you please check all of them?